### PR TITLE
Fixing pull pull request on themes

### PR DESCRIPTION
**Pull request by maxwellfundi**

### CODE REVIEW

**Comment by maxwellfundi** [comment]
@dannyparsons I hope this is now good with no errors

File: instat/sdgPlots.vb [hunk 1]
```diff
@@ -42,14 +42,76 @@ Public Class sdgPlots
         chkIncludeFacets.Checked = False
         FacetsCheck(False)
 
+        'setting current text on the ucrInputThemeList
+
+
+        ucrInputThemeLists.SetName("Choose Theme Here")
```
Comment (dannyparsons) [inline instat/sdgPlots.vb]: We don't want this inside the control. The control only contains themes. If you want to explain the control then use a label next to it.

**Comment by maxwellfundi** [comment]
This works but not very well. A couple of thing i am still to fix

**Comment by maxwellfundi** [comment]
I am closing this because i am redoing the themes.

**Pull request closed without merging**